topics/microbiome/tutorials/dada-16S/tutorial.md

```diff
@@ -0,0 +1,1050 @@
+>            - {% icon param-collection %} *"Paired short read data"*: `Raw Reads`
+>        - *"Aggregate data"*: `No`
+>    - *"sample number"*: `10000000`
+>
+> 2. Inspect forward read output
+{: .hands_on}
+</div>
+
+**dada2: plotQualityProfile** generates 2 files with quality profiles per sample: 1 PDF for forward reads and 1 PDF for reverse reads.
+
+> <hands-on-title>Inspect read quality profiles</hands-on-title>
+>
+> 1. Inspect **plotQualityProfile** forward read output
+{: .hands_on}
+
+![Quality Profiles for every samples with read length on the X-axis and Quality score on the Y-axis](./images/plotQualityProfile_forward.png)
+
+In gray-scale is a heatmap of the frequency of each quality score at each base position. The mean quality score at each position is shown by the green line, and the quartiles of the quality score distribution by the orange lines. The red line shows the scaled proportion of reads that extend to at least that position (this is more useful for other sequencing technologies, as Illumina reads are typically all the same length, hence the flat red line).
+
+> <question-title></question-title>
+>
+> 1. Are the forward reads good quality?
+> 2. We generally advise trimming the last few nucleotides to avoid less well-controlled errors that can arise there. What would you recommend as threshold?
+> 3. Are the reverse reads good quality?
+> 4. What would you recommend as threshold for trimming?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The forward reads are good quality
+> > 2. These quality profiles do not suggest that any additional trimming is needed. We will truncate the forward reads at position 240 (trimming the last 10 nucleotides).
+> > 3. The reverse reads are of significantly worse quality, especially at the end, which is common in Illumina sequencing. This isn’t too worrisome, as DADA2 incorporates quality information into its error model which makes the algorithm robust to lower quality sequence, but trimming as the average qualities crash will improve the algorithm’s sensitivity to rare sequence variants.
+> > 
+> >    ![Quality Profiles for every samples with read length on the X-axis and Quality score on the Y-axis](./images/plotQualityProfile_reverse.png)
+> >    
+> > 4. Based on the profiles, we will truncate the reverse reads at position 160 where the quality distribution crashes.
+> > 
+> {: .solution}
+>
+{: .question}
+
+> <comment-title>Considerations for your own data</comment-title>
+>
+> Your reads must still overlap after truncation in order to merge them later! The tutorial is using 2x250 V4 sequence data, so the forward and reverse reads almost completely overlap and our trimming can be completely guided by the quality scores. If you are using a less-overlapping primer set, like V1-V2 or V3-V4, your truncated length must be large enough to maintain $$ 20 + biological.length.variation $$ nucleotides of overlap between them.
```

Comment:
Can you add the info on how long V4 is? Then readers will know easier why the reads overlap.

Reply:
Done